### ISSUE TITLE
use sttm api to check for and display doodle

### ISSUE THREAD

**Issue by ManjotS** [KhalisFoundation/SikhiToTheMax-Web#422]
- [ ] utilize the private api to check for and display a doodle on home page.
- [ ] display last 10 doodles on sikhitothemax.org/doodle
- [ ] add short code for sttm.co/doodle

**Comment by ManjotS** [comment]
API now delivers doodles with width/height and any associated CSS that may be needed:

api.sikhitothemax.org/doodle/ will deliver:
`
{"rows":[{"ID":2,"ForDate":"2019-11-14","Year":2019,"MonthDay":1114,"Description":"something","Link":"http://sikhipedia.com","Image":"https://www.google.com/images/branding/googlelogo/2x/googlelogo_color_272x92dp.png","Height":92,"Width":272,"CSS":null,"Created":"2019-11-14 18:42:15"}]}
`

**Comment by tarunsingh5** [comment]
@ManjotS can we close this out now?

**Comment by ManjotS** [comment]
there are still 3 tasks remaining

**Comment by tarunsingh5** [comment]
@saintsoldierx thoughts?

**Comment by saintsoldierx** [comment]
yes, we do need an easier way to be able to upload doodles ourselves at the least.
